AutoCal/Controller/DashboardVC.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ViewController.swift
+//  AutoCal
+//
+//  Created by Ilan Harari on 4/6/20.
+//  Copyright © 2020 Ilan Harari. All rights reserved.
+//
+
+import UIKit
+import EventKit
+import EventKitUI
+
+class DashboardVC: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var reminderSelected : EKReminder!
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.remindersArray.count //count = 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = scheduledTable.dequeueReusableCell(withIdentifier: "reminderCell")!
+        cell.textLabel?.textColor = .white
+        cell.textLabel?.textAlignment = .center
+        cell.textLabel?.text = remindersArray[indexPath.item].title
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        reminderSelected = remindersArray[indexPath.row]
+    }
+    
+    @IBOutlet weak var scheduledTable: UITableView!
+    @IBOutlet weak var remindersLoading: UIActivityIndicatorView!
+    
+    var remindersArray: [EKReminder] = []
+    let remindersStore = EKEventStore()
+    
+    override func viewDidLoad() {
+        //MARK: View Constructor
+        super.viewDidLoad()
+        self.remindersLoading.isHidden = false
+        switch EKEventStore.authorizationStatus(for: .reminder) {
+        case .notDetermined:
+            self.remindersStore.requestAccess(to: .reminder, completion: {(granted: Bool, error: Error?) -> Void in
+                if granted {
+                print("Access granted")
```

Comment:
There appears to be a number of indentation inconsistencies in this file. You can fix them by selecting all then go to Editor -> Structure -> Re-indent